README.rst

```diff
@@ -67,3 +69,30 @@ via include list.
 This state will start the template service and has a depency on ``template.config``
 via include list.
 
+``template.clean``
+------------------
+
+Meta-state (This is a state that includes other states)
+
+This state will undo everything performed in the ``template`` meta-state in reverse order, i.e.
+stops the service,
+removes the configuration file and
+then uninstalls the package.
+
+``template.service.clean``
+--------------------------
+
+This state will stop the template service only.
```

Comment:
"This state will stop the service and disable it at boot time."

Reply:
No problem, will push now.